www/docs/faq.md

```diff
@@ -196,9 +196,9 @@ JSON Web Tokens can be used for session tokens, but are also used for lots of ot
 
   NextAuth.js client includes advanced features to mitigate the downsides of using shorter session expiry times on the user experience, including automatic session token rotation, optionally sending keep alive messages to prevent short lived sessions from expiring if there is an window or tab open, background re-validation, and automatic tab/window syncing that keeps sessions in sync across windows any time session state changes or a window or tab gains or loses focus.
 
-* As with database session tokens, JSON Web Tokens are limited in the amount of data you can store in them. There is typically a limit of around 4096 bytes in total for all cookies on a domain, though the exact limit varies between browsers, proxies and hosting services.
+* As with database session tokens, JSON Web Tokens are limited in the amount of data you can store in them. There is typically a limit of around 4096 bytes in total for a single domain's cookies,  though the exact limit varies between browsers, proxies and hosting services. If you want to support most browsers, then do not exceed 50 cookies per domain and do not exceed 4096 bytes per domain. (Source: [browsercookielimits.iain.guru](http://browsercookielimits.iain.guru/))
```

Comment:
4096 bytes per COOKIE, at least how I understand it. the important take here is that our session cookie can store up to 4096 bytes

Reply:
The `browsercookielimits` page says the following:

>If you want to support most browsers, then don't exceed 50 cookies per domain, and don't exceed 4093 bytes per domain (i.e. total size of all cookies <= 4093 bytes)

Reply:
Ah alright awesome, good eye! 

I'll adjust the text then shortly